app/v1/dist/index.html

```diff
@@ -225,6 +239,20 @@
           gui = new dat.GUI({ autoPlace: false });
 console.log(`QQ/gui/domElement: ${gui.domElement}`);
           viewerElement.appendChild(gui.domElement);
+          
+          //Add a download button to the page it TODO: this process will need to updated for downloading other file types like .pdf
+          
+          downloadButton = document.createElement("button");
+          downloadButton.setAttribute("type", "button");
+          downloadButton.appendChild(document.createTextNode("Download"));
+          downloadButton.setAttribute("style", "position: absolute; top: 2px; right: 2px;");
+          downloadButton.setAttribute("disabled", "true");
+          downloadButton.addEventListener("click", (e) => {
```

Comment:
If you don't use the (e) it's probably better to write () => { instead of (e) => {